keep/api/routes/preset.py

```diff
@@ -60,6 +64,51 @@
     ),
 }
 
+async def pull_alerts_from_providers(
+    tenant_id: str, pusher_client: Pusher | None, sync: bool = False
```

Comment:
redundant, isn't it?
```suggestion
    tenant_id: str
```

Reply:
or maybe send a "poll-alerts" (or is it happening afterwards?)

Reply:
looks like it just uses process_event which does the work :)

Reply:
<3